shared/js/ui/base/ui-wrapper.es6.js

```diff
@@ -58,6 +58,23 @@ const closePopup = () => {
     w.close()
 }
 
+const startBreakageFlow = (tabId, flowStyle, cb) => {
+    window.chrome.tabs.reload(tabId, { bypassCache: true }, () => {
+        setTimeout(() => {
+            window.chrome.tabs.executeScript(tabId, {
+                code: `window.breakageModel = '${flowStyle}'`
+            }, () => {
+                console.log('execute 2')
+                window.chrome.tabs.executeScript(tabId, {
+                    file: 'public/js/content-scripts/breakage-flow.js'
```

Comment:
Rather than injecting in a different script and also needing a window global, can you make this a `content-scope` script and use the `update` method as cookies use to send the `flowStyle`?

This will have the following advantages:
- Using remote config is built in by default
- There isn't an exposed global on the page that the page could potentially attack
- One unified script, dynamic script execution is potentially going soon with manifest v3

Let me know if you need some pointers on this code as I know it's a little more complex.